### ISSUE TITLE
Refactor target models

### ISSUE THREAD

**Issue by bentsherman** [ctargon/TSPG#7]
The three target model classes can be reduced to a single class with arguments to control hiddlen layer sizes and batch norm. Additionally, this code should be updated to use Keras API as the target model is a standard MLP.

**Comment by bentsherman** [comment]
Done in recent commits.
